tests/devices/test_default_qubit.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
Are there any changes actually done here other than formatting? I couldn't find any

Reply:
Ah, I just changed 2020 to 2021.